### PR TITLE
Update vnet-deployment.md

### PR DESCRIPTION
**Pull request by cosh**

### CODE REVIEW

**Comment by shsagir** [comment]
Docs Build status updates of commit _[c6ead8b](https://github.com/cosh/dataexplorer-docs/commits/c6ead8bebd5e7251b857ac8b373f0617d90e0030)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/vnet-deployment.md](https://github.com/cosh/dataexplorer-docs/blob/patch-4/data-explorer/vnet-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202208150638180700-1353/BuildReport?accessString=0920c6f994fc5c87cb5f5e198dd8fd8530271980476408d238dbed71d84cfd33).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by Jak-MS** [comment]
@shsagir - Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/vnet-deployment.md [hunk 1]
```diff
@@ -76,10 +76,13 @@ Create a [private endpoint](/azure/private-link/private-endpoint-overview) to re
 
 ### Configure Network Security Group rules using subnet delegation
 
-We recommend that you use [subnet delegation](/azure/virtual-network/subnet-delegation-overview) for your cluster's deployment. To do so, you must delegate the subnet to *Microsoft.Kusto/clusters* before creating the cluster in the subnet.
+Configuration of Network Security Group rules using [subnet delegation](/azure/virtual-network/subnet-delegation-overview) is the default for Virtual Network injected Azure Data Explorer clusters. To do so, you must delegate the subnet to *Microsoft.Kusto/clusters* before creating the cluster in the subnet.
```
Comment (shsagir) [inline data-explorer/vnet-deployment.md:79]:
```suggestion
[Subnet delegation](/azure/virtual-network/subnet-delegation-overview) is the default method for configuring Network Security Group rules for Azure Data Explorer clusters deployed into a subnet in your virtual network. When using subnet delegation, you must delegate the subnet to *Microsoft.Kusto/clusters* before creating the cluster in the subnet.
```

File: data-explorer/vnet-deployment.md [hunk 2]
```diff
@@ -76,10 +76,13 @@ Create a [private endpoint](/azure/private-link/private-endpoint-overview) to re
...
 
 By enabling subnet delegation on the cluster's subnet, you enable the service to define its pre-conditions for deployment in the form of Network Intent Policies. When creating the cluster in the subnet, the NSG configurations mentioned in the following sections are automatically created for you.
 
+ > [!WARNING]
+ > Your Azure Data Explorer cluster will stop working eventually if you change the subnet delegation configuration. This might mean that it will not be able to start after it was stopped or we will not be able to apply management or health monitoring on the cluster.
```
Comment (shsagir) [inline data-explorer/vnet-deployment.md:84]:
```suggestion
  > Changing your subnet delegation configuration will eventually disrupt the normal operation of your cluster. For example, after stopping the cluster you may not be able to start your cluster, run management commands, or apply health monitoring on your cluster.
```

Comment (shsagir) [reply data-explorer/vnet-deployment.md:84]: How do you fix remedy this situation?

**Review by shsagir — CHANGES_REQUESTED** [review]

**Comment by YaelSchuster** [comment]
@cosh please note Shlomo's question and respond so we can get this closed

**Comment by shsagir** [comment]
Docs Build status updates of commit _[073ef1b](https://github.com/cosh/dataexplorer-docs/commits/073ef1b58b5643dd24b6a38929a289daa3a3aa9f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/vnet-deployment.md](https://github.com/cosh/dataexplorer-docs/blob/patch-4/data-explorer/vnet-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202208300938019512-1353/BuildReport?accessString=f552766699993c4b082345c2aa12a94bf0c176eb1d6319167976655b7b44bd68).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by shsagir** [comment]
Docs Build status updates of commit _[68de88d](https://github.com/cosh/dataexplorer-docs/commits/68de88dedb5cfa1ed34c5b0bfc7079286c999097)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/vnet-deployment.md](https://github.com/cosh/dataexplorer-docs/blob/patch-4/data-explorer/vnet-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202208300939036583-1353/BuildReport?accessString=2357d7fb8443dff3df10a61df90ec2b4760e5335a67fd22d3e7167f395db3191).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by shsagir** [comment]
#sign-off

**Comment by shsagir** [comment]
Docs Build status updates of commit _[3f74e45](https://github.com/cosh/dataexplorer-docs/commits/3f74e457b02199d2ccc9b7b38f7e223892a9160c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/vnet-deployment.md](https://github.com/cosh/dataexplorer-docs/blob/patch-4/data-explorer/vnet-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202208311442044760-1353/BuildReport?accessString=c5cb4a88ca6e75b57ba61e09c6fe92dfbb0a2bd9ce7823b33521a12f1529cd49).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**